staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere.go

```diff
@@ -951,6 +955,20 @@ func (vs *VSphere) AttachDisk(vmDiskPath string, storagePolicyName string, nodeN
 		}
 	}
 	klog.V(4).Infof("AttachDisk executed for node %s and volume %s with diskUUID %s. Err: %s", convertToString(nodeName), vmDiskPath, diskUUID, err)
+	if err != nil {
```

Comment:
it would be good if we can move logic in this code block in the `if err != nil {` block written on top of this code block.

Reply:
it can go either way but I somewhat prefer the current style so I have left it as it is.